library/src/main/java/com/getstream/sdk/chat/view/AvatarView.kt

```diff
@@ -95,6 +95,7 @@ class AvatarView @JvmOverloads constructor(
 	}
 }
 
+private const val MIN_RADIO_SIZE = 100
```

Comment:
So, this is a default radius size, right? 

1.  Its value should be defined in density-independent size, otherwise, it will have different actual size on different screen's densities. 

2. I'd call it MIN_RADIUS_SIZE or DEFAULT_RADIUS_SIZE

Reply:
1-. It is not needed, in the end it is to create an image that will be rendered and resized on the drawing process according to with the Avatar size (can be changed on runtime)
2-. Yes, I have updated it ;)

Reply:
Alright, thanks for explanation :)